### ISSUE TITLE
Cover image tooltips for alignment are duplicated

### ISSUE THREAD

**Issue by karmatosed** [WordPress/gutenberg#6013]
The same alignment tooltip is used for both the image and text alignment, should this be the case?

![align](https://user-images.githubusercontent.com/253067/38373204-92586b58-38e8-11e8-8ec2-09e73368cacb.png)
![align2](https://user-images.githubusercontent.com/253067/38373205-928a870a-38e8-11e8-8391-ce5135c7593a.png)

**Comment by afercia** [comment]
This still happens on current master 3.8.0-rc.1 and applies also to align center, align right. The tooltips are generated based on the aria-label attributes and they need to be distinguished, to properly communicate assistive technology users what they're meant for.

**Comment by NickOrtiz** [comment]
Should the copy in these locations read "Text align left" and "Image align left" for the different tooltips?  Or is there more appropriate descriptive terminology we should be using?
